stan/math/prim/mat/fun/to_matrix.hpp

```diff
@@ -1,14 +1,19 @@
 #ifndef STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 #define STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 
+#include <boost/math/tools/promotion.hpp>
+#include <stan/math/prim/scal/err/check_size_match.hpp>
+#include <stan/math/prim/scal/err/invalid_argument.hpp>
 #include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <Eigen/Dense>
```

Comment:
The include on line 7 is all you need---it includes `Eigen/Dense` and makes sure everything's include din the right order.